apps/www/lib/apollo/miniNavi.js

```diff
@@ -15,8 +16,44 @@ export const miniNaviQuery = gql`
   }
 `
 
+let refetchTimeoutId
+
+const PUBLISH_HOURS = 5
+const PUBLISH_MINUTES = 0
+
 export const useFlyerMeta = () => {
-  const { data } = useQuery(miniNaviQuery)
+  const { data, refetch } = useQuery(miniNaviQuery)
+
+  useEffect(() => {
+    if (refetchTimeoutId !== undefined) {
+      return
+    }
+    const now = new Date()
+    const updateTimeToday = new Date(
+      now.getFullYear(),
+      now.getMonth(),
+      now.getDate(),
+      PUBLISH_HOURS,
+      PUBLISH_MINUTES,
+    )
+    const nextUpdateTomorrow = new Date(
+      now.getFullYear(),
+      now.getMonth(),
+      now.getDate() + 1,
+      PUBLISH_HOURS,
+      PUBLISH_MINUTES,
+    )
```

Comment:
I have doubts that this work as expected. Just adding +1 to the day would yield invalid dates at the end of every month.

I would compute the next date simply by adding 1 day in ms to the current timestamp.

(see [this fiddle](https://jsfiddle.net/78v2y9pa/1/) to see the issue)

```suggestion
    const DAY_IN_MS = 24 * 60 * 60 * 1000
    const nextUpdateTomorrow = new Date(updateTimeToday.getTime() + DAY_IN_MS)
```